src/executables/util_cs.c

```diff
@@ -4603,6 +4605,17 @@ memmon (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  if (finalize_force)
+    {
+      error_code = mmon_disable_force ();
+      if (error_code != NO_ERROR)
+	{
```

Comment:
I think we need to print error message here

Reply:
Done.